en/docs/get-started/tutorials/create-your-first-event-triggered-using-containerized-app.md

```diff
@@ -0,0 +1,125 @@
+# Create an Event-Triggered Integration as a BYOC component
+
+Event-triggered integrations can help automate business processes and reduce manual work. Choreo allows you to create and manage event-triggered integrations without complex infrastructure setup.
```

Comment:
If we are changing the tilte, we can perhaps change the intro as follows to align with it. Otherwise, this just seems like a repetition of the intro in the `Create Your First Event-Triggered Integration` tutorial.
Suggestion:
Event-triggered integrations can help automate business processes and workflows in response to specific events. With Choreo, you can deploy your containerized application as an event-triggered integration component, which makes your application deployment process efficient and reduces manual intervention.